Dockerfile

```diff
@@ -1,18 +1,16 @@
 FROM node:12.17.0-alpine
 
-ENV YARN_CACHE_FOLDER=/dev/shm/yarn-cache
-
 WORKDIR /app
 
 RUN apk --no-cache add --virtual native-deps \
   g++ gcc libgcc libstdc++ linux-headers make python git \
   chromium chromium-chromedriver
 
-COPY yarn.lock package.json /app/
-RUN yarn --dev --frozen-lockfile
+COPY package*.json /app/
+RUN npm install --development
```

Comment:
btw what are the dev deps used for?

Reply:
Dev dependencies are used in the build phase, now they are installed with npm instead of yarn